posts/views.py

```diff
@@ -82,8 +85,13 @@ def get_post(self, pk):
     # TODO: (<AUTHENTICATION>, <VISIBILITY>) check VISIBILITY before getting
     def get(self, request, pk):
         post = self.get_post(pk)
+        if post.contentType == "text/markdown":
+            post_content = commonmark.commonmark(post.content)
+        else:
+            post_content = "<p>" + post.content + "</p>"
+
         serializer = PostSerializer(post)
-        return Response(serializer.data)
+        return render(request, 'post/post.html', context={'post': serializer.data, 'post_content': post_content})
```

Comment:
If this is the api view why should it be returning a render instead of a response, and having frontend/post/<id> be using a frontendPostViewID view instead of PostViewID? Seems like this is out of whack

Reply:
This was from when I branched from kieter. I fixed this up. Good catch though